config/karma/launchers.js

```diff
@@ -0,0 +1,124 @@
+const customLaunchers = {
+  Safari: {
+    base: 'SafariNative',
+  },
```

Comment:
do you really need that file?

Reply:
To be fair we could drop everything related to karma here (it's not tested in ci). I only added them to know how it would behave before working on sdk-js